docs/source/posts/2023/2023-08-24-final-report-tvcastillod.rst

```diff
@@ -0,0 +1,175 @@
+   :target: https://summerofcode.withgoogle.com/programs/2023/projects/ymwnLwtT
+
+.. image:: https://www.python.org/static/community_logos/python-logo.png
+   :width: 40%
+   :target: https://summerofcode.withgoogle.com/programs/2023/organizations/python-software-foundation
+
+.. image:: https://python-gsoc.org/logos/FURY.png
+   :width: 25%
+   :target: https://fury.gl/latest/index.html
+
+Google Summer of Code Final Work Product
+========================================
+
+.. post:: August 24 2023
+   :author: Tania Castillo
+   :tags: google
+   :category: gsoc
+
+-  **Name:** Tania Castillo
+-  **Organisation:** Python Software Foundation
+-  **Sub-Organisation:** FURY
+-  **Project:** `SDF-based uncertainty representation for dMRI glyphs <https://github.com/fury-gl/fury/wiki/Google-Summer-of-Code-2023-(GSOC2023)#project-3-sdf-based-uncertainty-representation-for-dmri-glyphs>`_
+
+
+Abstract
+--------------------
+Diffusion Magnetic Resonance Imaging (dMRI) is a non-invasive imaging technique used by neuroscientists to measure the diffusion of water molecules in biological tissue. The directional information is reconstructed using either a Diffusion Tensor Imaging (DTI) or High Angular Resolution Diffusion Imaging (HARDI) based model, which is graphically represented as tensors and Orientation Distribution Functions (ODF). Traditional rendering engines discretize Tensor and ODF surfaces using triangles or quadrilateral polygons, making their visual quality depending on the number of polygons used to build the 3D mesh which might compromise real-time display performance. This project proposes a methodological approach to further improve the visualization of DTI tensors and HARDI ODFs glyphs by using well-established techniques in the field of computer graphics such as geometry amplification, billboarding, signed distance functions (SDFs), and ray marching.
+
+
+Proposed Objectives
+-------------------
+
+- Implement a parallelized version of computer-generated billboards using geometry shaders for amplification.
+- Model the mathematical functions that express the geometry of ellipsoids glyphs and implement them using Ray Marching techniques.
+- Model the mathematical functions that express the geometry of ODF glyphs and implement them using Ray Marching techniques.
+- Use SDF properties and techniques to represent the uncertainty of dMRI reconstruction models.
+
+
+Objectives Completed
+--------------------
+
+Ellipsoid actor implemented with SDF
+************************************
+
+A first approach for tensor glyph generation has been made, using raymarching and SDF applied to a box. The current implementation with ``tensor_slicer`` requires a sphere with a specific number of vertices to be deformed based on this model, to get a higher resolution a sphere with more vertices is needed. Because the raymarching technique does not use polygonal meshes it is possible to define perfectly smooth surfaces and still obtain a fast rendering.
```

Comment:
`based on this model, to get a highet (....)` -> `based on this model, but to get a higher (...)` and `polygonal meshes it is possible to` -> `polygonal meshes, it is possible to`